src/components/GeneralInfo/GeneralInformation/GeneralInformation.js

```diff
@@ -193,6 +193,16 @@ class GeneralInformation extends Component {
                     />
                   </GridItem>
                 )}
+
+                {this.props.showRuntimesProcesses && (
```

Comment:
Do all hosts have fqdn associated? and if not should we check `fqdn` presence as well not to load the module when it is unnecessary?

Reply:
Good idea, taking a quick look at the open api spec it looks like fqdn is not a required parameter in host inventory so a check sounds like a good safeguard.